app/src/full/java/io/homeassistant/companion/android/notifications/MessagingService.kt

```diff
@@ -681,7 +681,7 @@ class MessagingService : FirebaseMessagingService() {
         groupId: Int,
         data: Map<String, String>
     ) {
-        val actionUri = data["clickAction"]
+        val actionUri = if (!data["clickAction"].isNullOrEmpty()) data["clickAction"] else "/lovelace/0"
```

Comment:
Does just `/` work?  I ask because if you change the path in Home Assistant I'm not sure what this url will do, does it give a 404 or does it redirect.

Reply:
yup `/`  it does indeed work :)